packages/round-manager/src/features/round/ViewRoundResults/ViewRoundResults.tsx

```diff
@@ -61,14 +89,19 @@ export default function ViewRoundResults() {
 
   const onRecalculateResults = () => {
     // Logic for recalculating results goes here
+    // DEBUG: timeout for testing
```

Comment:
Another: 
What should actually happen here? 

Are we making a POST request to `/chains/:chainId/rounds/:roundId/matches`?

Reply:
yes!